dds_web/api/user.py

```diff
@@ -60,6 +66,88 @@ def jwt_token(username):
 ####################################################################################################
 
 
+class ConfirmInvite(flask_restful.Resource):
+    def get(self, token):
+        """ """
+
+        s = itsdangerous.URLSafeTimedSerializer(app.config.get("SECRET_KEY"))
+
+        try:
+            # Get email from token
+            email = s.loads(token, salt="email-confirm", max_age=10000)
+
+            # Get row from invite table
+            invite_row = models.Invite.query.filter(models.Invite.email == email).first()
+
+        except itsdangerous.exc.SignatureExpired:
+            raise
+        except sqlalchemy.exc.SQLAlchemyError as sqlerr:
+            raise DatabaseError(str(sqlerr))
+
+        # Check the invite exists
+        if not invite_row:
+            raise NoSuchInviteError(email=email)
+
+        # Get facility info from db
+        facility_name = None
+        if invite_row.is_facility:
+            try:
+                facility_info = (
+                    models.Facility.query.filter(models.Facility.id == invite_row.facility_id)
+                    .with_entities(models.Facility.name)
+                    .first()
+                )
+            except sqlalchemy.exc.SQLAlchemyError as sqlerr:
+                raise DatabaseError(str(sqlerr))
```

Comment:
Is this sqlerr only sent to the log or is it shown in the response as well? I guess it should not be shown to the end user?